tests/test_series.py

```diff
@@ -57,6 +59,75 @@ def csv_path(tmp_path):
     return csv_path
 
 
+def test_primary_prediction_not_found(tmp_path):
+    dummy_video_path = tmp_path / "dummy_video.mp4"
```

Comment:
Always use Path from pathlib when constructing paths to avoid errors different platforms. `Series.load` accepts paths since Path(Path(x))=Path(x) so you shouldn't need to convert to string later on